tabular/src/autogluon/tabular/models/tab_transformer/tab_transformer.py

```diff
@@ -0,0 +1,157 @@
+# Code in this script based on work by Milan Cvitkovic,
+# Xin Huang, Ashish Khetan and Zohar Karnin.
+
+import copy
+
+import torch
+import torch.nn as nn
+
+from .tab_model_base import TabModelBase
+
+
+class TabTransformer(TabModelBase):
+    """
+    Transformer model for tabular data
```

Comment:
add comment that it can be useful for semisupervised learning

Reply:
Added additional comment.